$NetBSD: patch-aa,v 1.11 2008/10/24 12:37:37 wiz Exp $

2008-10-23  Bruno Haible  <bruno@clisp.org>
        Define a dummy SA_NODEFER macro on Interix.
        * lib/signal.in.h (SA_NODEFER): Define fallback.
        Reported by Aleksey Cheusov <cheusov@tut.by> via
        Thomas Klausner <wiz@netbsd.org> and Eric Blake.

--- lib/signal.in.h.orig	2008-09-26 12:32:24.000000000 +0000
+++ lib/signal.in.h
@@ -176,6 +176,12 @@ extern int sigaction (int, const struct 
 #endif /* !@HAVE_SIGACTION@, !@HAVE_STRUCT_SIGACTION_SA_SIGACTION@ */
 
 
+/* Some systems don't have SA_NODEFER.  */
+#ifndef SA_NODEFER
+# define SA_NODEFER 0
+#endif
+
+
 #ifdef __cplusplus
 }
 #endif
